src/Numerics/LinearRegression/DemingRegression.cs

```diff
@@ -0,0 +1,203 @@
+﻿// <copyright file="DemingRegression.cs" company="Math.NET">
+// Math.NET Numerics, part of the Math.NET Project
+// http://numerics.mathdotnet.com
+// http://github.com/mathnet/mathnet-numerics
+//
+// Copyright (c) 2009-2023 Math.NET
+//
+// Permission is hereby granted, free of charge, to any person
+// obtaining a copy of this software and associated documentation
+// files (the "Software"), to deal in the Software without
+// restriction, including without limitation the rights to use,
+// copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following
+// conditions:
+//
+// The above copyright notice and this permission notice shall be
+// included in all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+// EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES
+// OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+// NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+// HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY,
+// WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+// OTHER DEALINGS IN THE SOFTWARE.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+
+namespace MathNet.Numerics.LinearRegression
+{
+    public static class DemingRegression
+    {
+        /// <summary>
+        /// Deming/Orthogonal regression, least-Squares fitting the points in
+        /// the 2D dataset (x,y) to a line
+        /// <code>
+        ///    a*x + b*y + c = 0
+        /// </code>
+        /// For <paramref name="delta"/> equal 1 (the default value), this is
+        /// performing orthogonal regression, minimizing the sum of squared
+        /// perpendicular distances from the data points to the regression line.
+        /// <para>
+        /// Orthogonal regression is a special case of Deming regression,
+        /// and is assuming equal error variances on the x and y data,
+        /// and applied by the argument <paramref name="delta"/> default value of 1.0.
+        /// </para>
+        /// <para>
+        /// The parameters (a,b,c) are scaled such that a and b
+        /// in absolute values are always less than one.
+        /// </para>
+        /// </summary>
+        /// <param name="x">X data</param>
+        /// <param name="y">Y data</param>
+        /// <param name="delta">Ratio of variances of x and y data, var(y)/var(x). Default value is 1</param>
+        /// <returns> returning its best fitting parameters as (a, b, c) tuple.</returns>
+        /// <remarks>
+        /// Solution algorithm as described from:
```

Comment:
I couldn't find the reference, only a wikipedia [linke](https://en.wikipedia.org/wiki/Deming_regression). Could you provide a pdf link or something similar?

Reply:
That was the best link I could find.